gallery/src/main.scss

```diff
@@ -6,6 +6,11 @@
  */
 @use 'sass:math';
 
+@font-face {
+  font-family: "Inconsolata";
```

Comment:
This is the font face that the gallery code examples specify.  We weren't providing it, so it turns out we've been getting a system-dependent font face there all along.  That creates differences between local snapshots and those on jenkins.

Reply:
> This is the font face that the gallery code examples specify. We weren't providing it, so it turns out we've been getting a system-dependent font face there all along. That creates differences between local snapshots and those on jenkins.

I'm not sure what's going on there but Courier Prime was selected as our monospace font here: https://issues.sonatype.org/browse/RSC-176?jql=text%20~%20%22monospace%22

Reply:
> I'm not sure what's going on there but Courier Prime was selected as our monospace font here:

The syntax highlighter themes have their own opinions about what font they want to use, which they set with no regard for our preferences.  The theme we had been using for a long time specified Inconsolata, but we hadn't been providing that font, and so the examples were actually rendering differently on different OSes/browsers.

That said, we have actually switched to a different syntax highlighter theme in the time since I wrote the above comment, and that new theme does not use Inconsolata, but rather specifies a handful of proprietary fonts.  That means it would delibreately look a bit different on different OSes, which isn't workable for visual tests, so I used a `!important` to override it back to Inconsolata.  Per your comment though, I probably should have overridden it to Courier Prime instead and gotten rid of Inconsolata since it's no longer what's desired either by the highlighter theme nor by us.  I will make that change

Reply:
a524a1c9acf4